README.md

```diff
@@ -1,24 +1,51 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Things you may want to cover:
+# Blog App
 
-* Ruby version
+This is a blog application that can have users write and post blogs, read blog posts, and edit or delete the posts.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby/Rails
+- PostgreSQL
 
-* Database creation
+## Getting Started
+Follow the steps bellow to get a local copy up and running.
 
-* Database initialization
+Run `git clone https://github.com/HENRYKC24/Hello-Rails.git` from your terminal to clone the repo from GitHub.
 
-* How to run the test suite
+Run `yarn install` from the command line to install dependencies.
```

Comment:
- [ ] I think you have missed some of the steps while writing the steps to run the project locally.

Like after cloning the repo, make sure you go to right folder, then add `username` and `password` according to your local postgresql setup, if you don't have please install it. 

```ruby
$ git clone https://github.com/HENRYKC24/Hello-Rails.git

$ cd Hello-Rails
$ bundle install

# Set username and password according to your postgresql in `config/database.yml`

# To create a database
$ rails db:create

# Below command is optional for this repo, but it is mandatory from the milestone where you will add the migration

$ rails db:migrate

# When you use seed data
$ rails db:seed

# then run server, which you have already mentioned 

$ rails s
```

Reply:
Thank you very much:pray:
Be sure to have me include that right away:running: